command/root.go

```diff
@@ -351,3 +352,39 @@ func determineEditor(cmd *cobra.Command) (string, error) {
 
 	return editorCommand, nil
 }
+
+func ExpandAlias(args []string) ([]string, error) {
+	empty := []string{}
+	if len(args) < 2 {
+		// the command is lacking a subcommand
+		return empty, nil
+	}
+
+	ctx := initContext()
+	cfg, err := ctx.Config()
+	if err != nil {
+		return empty, err
+	}
+	aliases, err := cfg.Aliases()
+	if err != nil {
+		return empty, err
+	}
+
+	if aliases.Exists(args[1]) {
+		expansion := aliases.Get(args[1])
+		for i, a := range args[2:] {
+			if !strings.Contains(expansion, "$") {
+				expansion += " " + a
```

Comment:
I'm wary of appending extra arguments via string concatenation. Because this string is going to be `shlex.Split` later, any single argument that contained spaces is going to be split into multiple arguments.

As a safer approach, could we collect extra (non-positional) arguments into a slice and then append that slice onto the result of `shlex.Split(expansion)`?